pack/tsk/tskp.json

```diff
@@ -395,6 +509,181 @@
+        "traits": "Talent. Science.",
+        "type_code": "asset",
+        "xp": 0
+    },
+    {
+        "code": "09042",
+        "cost": 3,
+        "customization_options": [
+            {
+                "xp": 0,
+                "text_change": "insert",
+                "position": 0,
+                "choice": "choose_card",
+                "quantity": 1,
+                "card": {
+                    "type": ["asset"],
+                    "trait": ["Tome", "Spell"]
+                }
+            },
+            {
+                "xp": 1,
+                "text_change": "append"
+            },
+            {
+                "xp": 1,
+                "text_change": "append"
+            },
+            {
+                "xp": 2,
+                "text_change": "append"
+            },
+            {
+                "xp": 2,
+                "text_change": "insert",
+                "position": 0,
+                "choice": "choose_card",
+                "quantity": 2,
+                "card": {
+                    "type": ["asset"],
+                    "trait": ["Tome", "Spell"]
+                }
+            },
+            {
+                "xp": 3,
+                "text_change": "append"
+            },
+            {
+                "xp": 4,
+                "text_change": "replace",
+                "position": 1
+            }
+        ],
+        "customization_change": "Named asset:\nUsing attached asset's [action] abilities does not provoke attacks of opportunity.\nAttached asset does not take up any slots.\nYou get +2 skill value while performing skill tests on attached asset.\nAdditional named assets:\n[fast] Exhaust The Raven Quill: Move 1 secret or charge from an asset you control to attached asset.\nAttach to a named asset you control, or you may search your deck, discard pile, and hand for a copy of a named asset and play it <i>(paying its cost)</i>. Then, attach The Raven Quill to it.",
+        "customization_text": "Named [[Tome]] or [[Spell]] asset:\n□ <b>Living Quill.</b> Using attached asset's [action] abilities does not provoke attacks of opportunity.\n□ <b>Spectral Binding.</b> Attached asset does not take up any slots.\n□□ <b>Mystic Vane.</b> You get +2 skill value while performing skill tests on attached asset.\n□□ <b>Endless Inkwell.</b> Name two more [[Tome]] or [[Spell]] assets:\n□□ <b>Energy Sap.</b> The Raven Quill gains: \"[fast] Exhaust The Raven Quill: Move 1 secret or charge from an asset you control to attached asset.\"\n□□□□ <b>Supernatural Record.</b> When you play The Raven Quill, instead of attaching it to a named asset you control, you may search your deck, discard pile, and hand for a copy of a named asset and play it <i>(paying its cost)</i>. Then, attach The Raven Quill to it.",
```

Comment:
```suggestion
        "customization_text": "Named [[Tome]] or [[Spell]] asset:\n□ <b>Living Quill.</b> Using attached asset's [action] abilities does not provoke attacks of opportunity.\n□ <b>Spectral Binding.</b> Attached asset does not take up any slots.\n□□ <b>Mystic Vane.</b> You get +2 skill value while performing skill tests on attached asset.\n□□ <b>Endless Inkwell.</b> Name two more [[Tome]] or [[Spell]] assets:\n□□ <b>Energy Sap.</b> The Raven Quill gains: \"[fast] Exhaust The Raven Quill: Move 1 secret or charge from an asset you control to attached asset.\"\n□□□ <b>Interwoven Ink.</b> After you resolve an [action] ability on attached asset, you may exhaust The Raven Quill to ready another asset you control.\n□□□□ <b>Supernatural Record.</b> When you play The Raven Quill, instead of attaching it to a named asset you control, you may search your deck, discard pile, and hand for a copy of a named asset and play it <i>(paying its cost)</i>. Then, attach The Raven Quill to it.",
```